mintapi/api.py

```diff
@@ -846,6 +851,18 @@ def get_invests_json(self):
         else:
             logger.error("FAIL2")
 
+    def get_investment_data_new(self):
+        investments = self.call_investments_endpoint()["Investment"]
+        for i in investments:
+            i.pop("metaData", None)
```

Comment:
Why does the metaData need to be removed?

Reply:
I reviewed the metaData object again and decided that the "lastUpdatedDate" field would actually be useful to flatten.  Everything else isn't useful information and by including it, we lose the ability to send investment data to a CSV.